### PR TITLE
Alignment the elements for the navbar

### PR DESCRIPTION
**Pull request by sarovin**
I think that the elements are not well aligned now.
This fix it for me. Do you agree?

### CODE REVIEW

File: src/NavBar.js [hunk 1]
```diff
@@ -47,15 +47,29 @@ const styles = StyleSheet.create({
...
-    height: Platform.OS === 'ios' || Platform.Version > 19 ? 64 : 44,
+    ...Platform.select({
+      ios: {
+        height: 64,
+      },
+      android: {
+        height: 64,
+      },
+    }),
     right: 0,
```
Comment (aksonov) [inline src/NavBar.js]: Are you sure that height should be the same for android/ios? We had issue with that...

**Comment by sarovin** [comment]
There is a problem with the Translucent StatusBar.
We can't see if the translucent is set in StatusBar, no?

schermata 2016-06-15 alle 15 55 54

**Comment by aksonov** [comment]
Okey, will you deal with possible issues after your PR ? ;)

**Comment by sarovin** [comment]
Yes, no problem.

**Pull request merged**